src/EditorFeatures/Core/Host/IStreamingFindReferencesPresenter.cs

```diff
@@ -32,17 +32,16 @@ internal interface IStreamingFindUsagesPresenter
         /// It can also show messages about no references being found at the end of the search.
         /// If false, the presenter will not group by definitions, and will show the definition
         /// items in isolation.</param>
-        FindUsagesContext StartSearch(string title, bool supportsReferences);
+        /// <param name="cancellationToken">External cancellation token controlling whether finding shoudl be canceled
+        /// or not.  This will be combined with a cancellation token owned by the <see cref="FindUsagesContext"/>.
```

Comment:
```suggestion
        /// <param name="cancellationToken">External cancellation token controlling whether finding should be canceled.
        /// This will be combined with a cancellation token owned by the <see cref="FindUsagesContext"/>.
```